pkgs/checks/lib/src/extensions/math.dart

```diff
@@ -5,21 +5,128 @@
 import 'package:checks/context.dart';
 
 extension NumChecks on Check<num> {
+  /// Expects that this number is greater than [other].
   void operator >(num other) {
     context.expect(() => ['is greater than ${literal(other)}'], (actual) {
       if (actual > other) return null;
       return Rejection(
           actual: literal(actual),
-          which: ['Is not greater than ${literal(other)}']);
+          which: ['is not greater than ${literal(other)}']);
     });
   }
 
+  /// Expects that this number is greater than or equal to [other].
+  void operator >=(num other) {
+    context.expect(() => ['is greater than ${literal(other)}'], (actual) {
```

Comment:
nit: greater than or equal to (same below)

Reply:
Done.